std/os/test.zig

```diff
@@ -0,0 +1,12 @@
+const std = @import("../index.zig");
```

Comment:
we need to make sure these tests get run when we run the stdlib tests. There is this code inside std/os/index.zig:

```zig
test "std.os" {
    _ = @import("child_process.zig");
    _ = @import("darwin_errno.zig");
    _ = @import("darwin.zig");
    _ = @import("get_user_id.zig");
    _ = @import("linux/errno.zig");
    //_ = @import("linux_i386.zig");
    _ = @import("linux/x86_64.zig");
    _ = @import("linux/index.zig");
    _ = @import("path.zig");
    _ = @import("windows/index.zig");
}
```

we need to add to this:

```zig
_ = @import("test.zig");
```

this way the test runs when you do:

`zig test std/index.zig`

and this will make sure travis ci and appveyor run the test.